acquia_lift.module

```diff
@@ -1,7 +1,6 @@
 <?php
 
 /**
- * @file
```

Comment:
Why removal of @file? https://www.drupal.org/coding-standards/docs#file still says it is valid.